internal/server/iifo_test.go

```diff
@@ -11,8 +11,10 @@ import (
 
 type mockClientIIFO struct {
 	baseclient.Client
-	foragingValues  shared.ForageShareInfo
-	otherIslandInfo []shared.ForageShareInfo
+	foragingValues                shared.ForageShareInfo
+	otherIslandInfo               []shared.ForageShareInfo
+	ownPrediction                 shared.PredictionInfo
```

Comment:
linter issue ownPrediction is unused apparently

Reply:
True didnt need it in the end.